discord/guild.py

```diff
@@ -1823,6 +1834,16 @@ async def edit(
         if premium_progress_bar_enabled is not MISSING:
             fields["premium_progress_bar_enabled"] = premium_progress_bar_enabled
 
+        if disable_invites is not MISSING:
+            if disable_invites:
+                if not "INVITES_DISABLED" in self.features:
+                    self.features.append("INVITES_DISABLED")
+            else:
+                if "INVITES_DISABLED" in self.features:
+                    self.features.remove("INVITES_DISABLED")
+
+            fields["features"] = self.features
```

Comment:
you should not be using `self.features` for this, instead make a new list similar to the `community` feature above
ideally it'd look something like this:
```py
features = []
if ...:
    features.append(...)
if ...:
    features.append(...)
if features:
    field["features"] = features
```

Reply:
thanks plun <3

Reply:
I tried similar code like this, but then I had the problem that features like community got also disabled because the features list got overwritten.

Reply:
mhh that might be an issue at our side.
discord fucks around if you transmit the stuff wrong.

Reply:
try making a copy of the guild's feature list (e.g. `list(self.features)`) and modifying that

Reply:
Code updated, did you mean like this?

Reply:
yeah but more similar to my example, so it includes the community guild feature as well

Reply:
Oh okay I will change it. Sry btw I´m still new in this kind of libary development as you may already noticed.

I tested it with this code and the bot didn´t disable the community feature. So seems to work or sould I still update it like your example?

Reply:
No need to apologize, everyone has their first steps at some point!

Reply:
Because If I understand that correct the feature list already includes the ``COMMUNITY`` Flag if its enabled. So idk no point of adding the flag again?

Reply:
> I tested it with this code and the bot didn´t disable the community feature. So seems to work or sould I still update it like your example?

yes, or we will have the same issue except with `COMMUNITY` being edited

Reply:
Ahh now I understand, updated now :)